paasta_tools/cli/cmds/spark_run.py

```diff
@@ -330,14 +330,35 @@ def get_docker_run_cmd(
     return cmd
 
 
+def get_default_event_log_dir(access_key, secret_key):
+    with open(DEFAULT_SPARK_RUN_CONFIG) as fp:
+        spark_run_conf = YAML().load(fp.read())
+    try:
+        account_id = boto3.client(
+            'sts',
+            aws_access_key_id=access_key,
+            aws_secret_access_key=secret_key,
+        ).get_caller_identity().get('Account')
+    except Exception:
+        account_id = None
+
+    for conf in spark_run_conf['environments'].values():
+        if account_id == conf['account_id']:
```

Comment:
Can this condition ever be true if `account_id` is `None` after the exception above?
If it can't then it's better to `return None` from the exception handler above instead of doing this loop.